book/src/02_core_ledger/07_specs/network.md

```diff
@@ -1,17 +1,17 @@
 # Network
 
-> Bringing Ouroboros to the people
+> Bringing Ouroboros to the people
 
 ---
 
-# Introduction
+## Introduction
```

Comment:
Hmm... I wonder if we could enforce "1 H1 heading per file"... Perhaps it's not worth the effort, just seems nice to have

Reply:
it is,  this change was because markdown lint complains if you have more than 1 H1 per file.  It also complains if you go from H2 to H4 and skip H3.